SU2_CFD/src/drivers/CDiscAdjMultizoneDriver.cpp

```diff
@@ -247,11 +243,6 @@ void CDiscAdjMultizoneDriver::Run() {
 
       config_container[iZone]->Set_StartTime(SU2_MPI::Wtime());
 
-      if (retape) {
-        SetRecording(NONE, Kind_Tape::FULL_TAPE, ZONE_0);
-        SetRecording(SOLUTION_VARIABLES, Kind_Tape::ZONE_SPECIFIC_TAPE, iZone);
-      }
```

Comment:
@oleburghardt I got rid of the zone-specific recording, the secondary recording still records all zones at once so the peak memory is the same and this way we have one less feature to maintain / debug.